shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
Yes, I saw it works for you. And I am trying to find the relationship for `only s390x ` on machines.cfg with.
I can not find any about s390 parameter setting anymore, maybe it should be same with some name like guest_os,cfg? 
`                    - 16.04-server.s390:
                        image_name += -16.04-server-s390
                        vm_arch_name = s390x
                        os_variant = ubuntutrusty
                        unattended_install, svirt_install:
                            kernel = images/ubuntu-server-16-04-s390/vmlinuz
                            initrd = images/ubuntu-server-16-04-s390/initrd.gz
                            boot_path = install
                        unattended_install.cdrom, svirt_install:
                            unattended_file = unattended/Ubuntu-16-04.preseed
                            cdrom_cd1 = isos/linux/ubuntu-16.04-server-s390.iso
                            md5sum_cd1 = 01545fa976c8367b4f0d59169ac4866c
                            md5sum_1m_cd1 = 4982356696dd141a340798a22bbf36e1`